### ISSUE TITLE
Unnecessary configuration step 

### ISSUE THREAD

**Issue by nelsonic** [dwyl/elixir-auth-google#19]
At present the instructions in the `README.md` include a step 
that requires people to add `config :elixir_auth_google` in their project's `config.exs` file:

![config-unnecessary](https://user-images.githubusercontent.com/194400/70195632-2666d880-16fe-11ea-8398-bb77e4c018dc.png)

I feel this is an _unnecessary_ step and should be defined in the _package's_ `config.exs` file.
We can reduce the number of steps for getting started _using_ the package by having [sensible defaults](https://en.wikipedia.org/wiki/Convention_over_configuration)
and if people decided they want to _override_ the `google_scope` they can do it _easily_.

```elixir
config :elixir_auth_google,
  google_scope: "profile email"
```
My _hypothesis_ is that _most_ people will use the defaults and never _need_ to change them.
_Forcing_ people to have to add lines to their project's `config.exs` [wastes people's time](https://github.com/dwyl/start-here/blob/master/manifesto.md#your-time-should-never-be-wasted).

**Comment by nelsonic** [comment]
@SimonLab thanks for sense checking this and merging the PR #21

**Comment by SimonLab** [comment]
Using the new version of the package on https://github.com/dwyl/app-mvp-phoenix didn't work as the client_id is not defined.
I think the package is trying to access the environment variable when they are not defined yet.
So we might still the step and the uses will need to define the configuration manually

**Comment by SimonLab** [comment]
It's possible that the configuration is not set because the `config` folder is not added to the hex package:
https://github.com/dwyl/elixir-auth-google/blob/3d7fa5805f440f7b0910fa4a5221b7374cfdb65f/mix.exs#L53

- [ ] Adding the config file to the mix config

**Comment by SimonLab** [comment]
Unfortunately adding the configuration file to the package doesn't seen to have resolve the issue. Moreover it seems that library configuration are not recommended:
https://hexdocs.pm/elixir/master/library-guidelines.html#avoid-application-configuration

Also the configuration of a library is not anymore used on the main application: https://elixirforum.com/t/how-to-access-config-key-in-package/1201/26

see also https://elixirforum.com/t/on-configuration-can-we-improve-how-elixir-libraries-applications-are-configured/18603

We could try to use `System.get_env` instead of `Application.get_env`

**Comment by nelsonic** [comment]
@SimonLab thanks for doing the additional research and fixing it with the PR: #24

**Comment by nelsonic** [comment]
Package published to https://hex.pm/packages/elixir_auth_google/1.0.2 👍

**Comment by nelsonic** [comment]
Resolved by #24 ✅
